# Andrew M. Frank

[andrew.m.frank@gmail.com][email] • [linkedin.com/in/andrewmfrank][linkedin] • [andrewfrank.github.io][github]

## Education

### University of Connecticut

Storrs, Connecticut  
Master of Science, Ecology and Evolutionary Biology  
2013-2018  

### American University

Washington, District of Columbia  
Bachelor of Science, Biology, *magna cum laude*, University Honors  
2007-2011  

## Technical Skills

- Designing cloud services in Google Cloud, using Compute Engine, Cloud Storage, Life Sciences, Batch, Cloud Run, Vertex AI, and BigQuery
- Managing cloud deployments using a GitOps approach; Terraform with CICD workflows
- Version control and project management with git, GitHub, and Cloud Source Repositories
- Deploying open-source, containerized, and reproducible bioinformatics tools and pipelines for genomics, metagenomics, transcriptomics, data processing, and statistical analysis; emphasis on deploying workflows with Nextflow and nf-core series community workflows
- Scripting in R and Bash, including in notebook environments; basic familiarity with Python, SQL, and YAML
- Working in Linux (strongly preferred), macOS, and Windows environments

## Professional Experience

### The Henry M. Jackson Foundation (HJF) — Bethesda, MD

IT Architect I  
March 2024 - Present  
- Lead the Uniformed Services University of the Health Sciences "Academic Research Cloud Initiative", a cloud environment for using and building research applications in the cloud
- Lorem ipsum

Bioinformatics Research Associate II  
November 2021 - February 2024  

- Led the operations of the "Student Bioinformatics Initiative" at the USUHS School of Medicine, a bioinformatics best-practices clearinghouse, cloud computing lab, and consultation center for bioinformatics users at USUHS
- Consulted on a portfolio of 20+ bioinformatics projects as part of SBI (50+ total projects to date), including assistance implementing GCP solutions for bioinformatics pipelines for WGS + RNASeq datasets, microbiome metagenomics, qPCR, querying large-scale tabular data, downstream statistical analyses, and automation of long-running laboratory data processing steps
- Managed SBI's GCP environment, including project creation, billing and usage monitoring, and proposing actions to ensure greater cloud spend efficiencies
- Authored 8 software tutorials to enable more efficient researcher use of SBI's cloud environment, including in-depth explorations of key GCP services, deploying the bioinformatics workflow orchestrator Nextflow, downstream statistical analyses in JupyterLab environments, and version control with Git
- Hosted and presented 20+ seminars for bioinformatics users at USUHS, with an emphasis on teaching the conceptual underpinnings of bioinformatics tools and processes, increasing the efficiency of cloud usage, and showcasing SBI-participant research outcomes
- Selected as key personnel to continue work at USUHS as part of HJF after expiration of the previous contract under TIAG

### The Informatics Applications Group (TIAG) — Reston, Virginia

Bioinformatics Scientist  
November 2020 - October 2021  

- Founded the "Student Bioinformatics Initiative" at the Uniformed Services University of the Health Sciences as part of a collaborative effort between multiple university entities to bring the power of cloud computing to graduate students and postdoctoral researchers
- Coordinated between various university stakeholders to ensure compliance and optimization of SBI’s cloud computing infrastructure and learning content
- Created and curated bioinformatics learning content for users, including bioinformatics tutorials, running virtual seminars, and a new university intranet site linking users to bioinformatics services across the university
- Advised on the creation of cloud infrastructure for the university; authored a demo server-less pipeline for handling PHI-containing data sources using GCP Workflows with the Healthcare and Data Loss Prevention APIs

### The American Type Culture Collection (ATCC) — Manassas, Virginia

Bioinformatician  
May 2018 - November 2020  

- Created and operated the back-end data management system for the ATCC Genome Portal, and ATCC database of 700+ bacterial, viral, and fungal reference genomes tied to specific authenticated material; using Python, SQL, AWS EC2 instances and AWS S3 buckets to manage data from Illumina and Oxford Nanopore platforms
- Developed all automated data processing pipelines and reports for BEI Resources, ATCC’s key government contract; handled de novo genome assembly, read mapping, and variant calling for DNASeq, RNASeq, metagenomic, and amplicon sequencing datasets
- Managed bioinformatics project goals for the ATCC Genome Portal in collaboration with partner organizations, tracked and ensured project goals were met on deadline, and reviewed data generated by partner bioinformaticians
- Interviewed, hired, and mentored a new bioinformatics internship position, who was subsequently hired as ATCC’s second full time bioinformatician
- Represented and communicated project results to the scientific community via publications, blog posts, presentations, and posters at scientific conferences

## Other Professional Experience

### University of Connecticut — Storrs, Connecticut

Bioinformatics Intern, Computational Biology Core  
January 2018 - May 2018  
  
National Science Foundation Graduate Research Fellow  
July 2014 - June 2017  

### Hartford Public Schools and Teach for America — Hartford, Connecticut

Science Teacher, High School  
June 2011 - May 2013  

## Publications

- Daley, B., *et al.* (including **Frank, A.**). (2024). SOS1 Inhibition Enhances the Efficacy of KRASG12C Inhibitors and Delays Resistance in Lung Adenocarcinoma. *Cancer Research*. <https://doi.org/10.1158/0008-5472.CAN-23-3256>
- Horseman, T.S., **Frank, A.**, *et al.* (2024). Effects of combined ciprofloxacin and Neulasta therapy on intestinal pathology and gut microbiota after high-dose irradiation in mice. *Frontiers in Public Health*. <https://doi.org/10.3389/fpubh.2024.1365161>
- Horseman, T.S., *et al.* (including **Frank, A.**). (2024). Ferroptosis, Inflammation, and Microbiome Alterations in the Intestine in the Göttingen Minipig Model of Hematopoietic-Acute Radiation Syndrome. *International Journal of Molecular Sciences*. <https://doi.org/10.3390/ijms25084535>
- Horseman, T.S., **Frank, A.**, *et al.* (2023). Longitudinal Changes in Lingual and Intestinal Microbiota in a Swine Burn Model. *Journal of Burn Care & Research*. <https://doi.org/10.1093/jbcr/irad045.075>
- Horseman, T.S., **Frank, A.**, *et al.* (2023). Meta-Analysis of Publicly Available Clinical and Preclinical Microbiome Data From Studies of Burn Injury. *Journal of Burn Care & Research*. <https://doi.org/10.1093/jbcr/irad098>
- Foreman, H., **Frank, A.**, Stedman T.T. (2021). Determination of variable region sequences form hybridoma immunoglobulins that target *Mycobacterium tuberculosis* virulence factors. *PLOS One*. <https://doi.org/10.1371/journal.pone.0256079>
- Riojas, M. and **Frank, A.** et al. (2020). A Rare Deletion in SARS-CoV-2 ORF6 Dramatically Alters the Predicted Three-Dimensional Structure of the Resultant Protein. *bioRxiv*. <https://doi.org/10.1101/2020.06.09.134460>
- **Frank, A.** (2019). The ATCC Genome Portal: Our Approach to Whole-genome Sequencing. *ATCC Technical Document*. <https://www.atcc.org/resources/technical-documents/genome-portal-technical-document>
- Lopera, J., **Frank, A.**, *et al.* (2019). Microbial Genome Databases: Existing Challenges and the Need for Authenticated Reference Genomes. *ATCC Application Note*. <https://www.atcc.org/resources/application-notes/microbial-genome-databases>

[email]: mailto:andrew.m.frank@gmail.com
[linkedin]: https://linkedin.com/in/andrewmfrank/
[github]: https://andrewfrank.github.io/
